python/test/hail/linalg/test_linalg.py

```diff
@@ -73,6 +76,23 @@ def check(expr, mean_impute, center, normalize, expected):
         with self.assertRaises(Exception):
             BlockMatrix.from_entry_expr(mt.x)
 
+    def test_write_from_entry_expr_overwrite(self):
+        mt = hl.balding_nichols_model(1, 1, 1)
+        mt = mt.select_entries(x=mt.GT.n_alt_alleles())
+
+        path = new_temp_file()
+        BlockMatrix.write_from_entry_expr(mt.x, path)
+        self.assertRaises(FatalError, lambda: BlockMatrix.write_from_entry_expr(mt.x, path))
+
+        BlockMatrix.write_from_entry_expr(mt.x, path, overwrite=True)
```

Comment:
can we also verify that reading after the overwrite produces a sam matrix?

Reply:
"a sam" = "the same"?

Reply:
erm yep